package.json

```diff
@@ -4,5 +4,12 @@
   "version": "4.4.0",
   "license": "MPL-2.0 AND MIT",
   "repository": "dequelabs/axe-core-nuget",
-  "private": true
+  "private": true,
+  "scripts": {
+      "serveFixtures": "cd node_modules/axe-test-fixtures/fixtures && http-server -s"
```

Comment:
Some of the new RunPartial tests assume that a root level npm install has already run before building the selenium/test project, and also that this `serveFixtures` script is running in the background already. This is why there are several `OneTimeSetUp: System.IO.DirectoryNotFoundException` errors, which are in turn the root cause of several `NullReferenceException`s during the tests.

I think it would be preferable to make the tests self-contained by:

* Moving these new `package.json` elements to a separate `package.json` file in `packages/selenium/test`
* Adding a build step to selenium/test that runs npm install, similar to the one in commons/src
* Adding a test setup/cleanup step that starts/stops the test server